components/support/rc_crypto/src/rand.rs

```diff
@@ -0,0 +1,26 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+use crate::{
+    error::*,
+    util::{ensure_nss_initialized, map_nss_secstatus},
+};
+
+pub fn fill(dest: &mut [u8]) -> Result<()> {
```

Comment:
I think it would be worth a comment here reassuring the casual reader the `NSS_Init` initializes the RNG with data for `/dev/urandom` , so we don't need to do any explicit initialization of the RNG.

Reply:
(Although I'm a little concerned to discover comments [like this](https://bugzilla.mozilla.org/show_bug.cgi?id=356886#c3) that suggest initialization behaviour can depend on the way NSS was built)